### PR TITLE
Update curve docs for tension

### LINKED ISSUE #2506: Curve tension
Hi guys! 
What is "curve tension" and how to change it?
CurveTension

### PR DESCRIPTION
**Pull request by slimbuck**
Fixes #2506

I confirm I have signed the [Contributor License Agreement](https://docs.google.com/a/playcanvas.com/forms/d/1Ih69zQfJG-QDLIEpHr6CsaAs6fPORNOVnMv5nuo0cjk/viewform).

### CODE REVIEW

File: src/math/curve.js [hunk 1]
```diff
@@ -20,7 +20,8 @@ import { CurveEvaluator } from './curve-evaluator.js';
  * * {@link pc.CURVE_STEP}
  *
  * Defaults to {@link pc.CURVE_SMOOTHSTEP}.
- *
+ * @property {number} tension The tension used when type is CURVE_SPLINE.
```
Comment (yaustar) [inline src/math/curve.js:23]: What is tension?

**Review by mvaligursky — APPROVED** [review]

File: src/math/curve.js (hunk 1 above)
Comment (slimbuck) [reply src/math/curve.js:23]: Lol. I wondered whether to link to the wikipedia entry? https://en.wikipedia.org/wiki/Cubic_Hermite_spline#Cardinal_spline

Comment (willeastcott) [reply src/math/curve.js:23]: That's fine to do that if you think the developer would find it informative. I normally find a lot of Wikipedia math explanations a bit overcomplicated, so a nice easy-to-understand summary of the meaning can be nice to have in the docs. Up to you.

Comment (yaustar) [reply src/math/curve.js:23]: Having it clear that it should be between 0 and 1 and a value of 1 means a straight line graph (does it? I'm really not sure 😅 )

Comment (slimbuck) [reply src/math/curve.js]: done

**Review by willeastcott — APPROVED** [review]

**Review by yaustar — APPROVED** [review]

**Pull request merged**